Datamigratie.Server/Features/Mapping/GlobalMapping/GetOverview/GetGlobalMappingOverviewController.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Datamigratie.Data;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+namespace Datamigratie.Server.Features.Mapping.GlobalMapping.GetOverview;
+
+[ApiController]
+[Route("api/[controller]")]
```

Comment:
controller naming should not be used since:
- renaming the the name of the controller should not break the API contract. 
- the path should not include the HTTP verb 
- it's against the URL naming convention   

```suggestion
[Route("api/global-mapping")]
```